docs/network/ble/btshell/btshell_GAP.md

```diff
@@ -0,0 +1,111 @@
+    - A device in the discoverable mode shall be in either the general discoverable mode or the limited discoverable mode.
+    - A device in non-discoverable mode will not be discovered by any device that is performing either the general discovery procedure or the limited discovery procedure.
+3. **Connection modes and procedures**
+    - allow a device to establish a connection to another device.
+    - allow updating of parameters of the connection 
+    - allow termination of the connection 
+4. **Bonding modes and procedures**
+    - Bonding allows two connected devices to exchange and store security and identity information to create a trusted relationship. 
+    - Bonding can occur only between two devices in bondable mode.
+
+
+<br>
+
+
+###Usage API
+
+
+|**Item No.** | **Modes and Procedures** | **nimBLE command** |
+|----|---------|---------------|
+|  1 | Broadcast Mode | `advertise conn=non discov=x` |
+|    | Observation Procedure | `scan duration=x passive=x filter=x` |
+|  2 | Non-Discoverable mode   | `advertise conn=x discov=non`  |
+|    | Limited Discoverable mode   | `advertise conn=x discov=ltd` |
+|    | General Discoverable mode   | `advertise conn=x discov=gen`  |
+|    | Limited Discovery procedure | `scan duration=x discov=ltd passive=0 filter=no_wl` |
+|    | General Discovery procedure | `scan duration=x discov=gen passive=0 filter=no_wl` |
+|    | Name Discovery procedure  | `scan duration=x` <br> `scan cancel` <br> `connect peer_addr_type=x peer_addr=x` <br> `gatt-read conn=x uuid=0x2a00` |
+|  3 | Non-connectable mode   | `advertise conn=non discov=x`  |
+|    | Directed connectable mode  | `advertise conn=dir [own_addr_type=x] [discov=x] [duration=x]`  |
+|    | Undirected connectable mode  | `advertise conn=und [own_addr_type=x] [discov=x] [duration=x]`  |
+|    | Auto connection establishment procedure   | `white-list addr_type=x addr=x [addr_type=y addr=y] [...]` <br> `connect addr_type=wl`  |
+|    | General connection establishment procedure  | `scan duration=x` <br> `scan cancel` <br> `connect peer_addr_type=x peer_addr=x` |
+|    | Selective connection establishment procedure | `white-list addr_type=x addr=x [addr_type=y addr=y] [...]` <br> `scan filter=use_wl duration=x` <br> `scan cancel` <br> `connect peer_addr_type=x peer_addr=x [own_addr_type=x]` |
+|    | Direct connection establishment procedure  | `connect addr_type=x addr=x [params]`  |
+|    | Connection parameter update procedure   | `conn-update-params conn=x <params>` |
+|    | Terminate connection procedure  | `disconnect conn=x` |
+|  4 | Non-Bondable mode    | `security-set-data bonding=0` [\*] |
+|    | Bondable mode        | `security-set-data bonding=1` [\*] |
+|    | Bonding procedure    | `security-start conn=x` [\*] |
+
+**[\*]** Security is disabled by default in bletiny.  To use the bonding modes and procedures, add `BLE_SM_LEGACY: 1` or `BLE_SM_SC: 1` to your syscfg.yml file depending on your needs.
```

Comment:
Change "Security is disabled by default in **bletiny**" to "Security is disabled by default in **btshell**"